test/unit/ConfigurationTest.php

```diff
@@ -306,4 +313,60 @@ public function getValidatorShouldReturnTheConfiguredValidator(): void
 
         self::assertSame($this->validator, $config->getValidator());
     }
+
+    /**
+     * @test
+     *
+     * @covers \Lcobucci\JWT\Configuration::getValidationConstraints
+     *
+     * @uses \Lcobucci\JWT\Configuration::forUnsecuredSigner
+     * @uses \Lcobucci\JWT\Configuration::__construct
+     * @uses \Lcobucci\JWT\Signer\None
+     * @uses \Lcobucci\JWT\Signer\Key
+     */
+    public function getValidationConstraintsShouldReturnTheDefaultWhenItWasNotConfigured(): void
+    {
+        $config                = Configuration::forUnsecuredSigner();
+        $validationConstraints = $config->getValidationConstraints();
+
+        self::assertSame([], $validationConstraints);
+        self::assertNotSame($this->validationConstraints, $validationConstraints);
```

Comment:
This assertion is not necessary since the one above already validates if the value is exactly `[]`, so I'd remove it.

Reply:
Which would allow us to inline the variable `$ValidationConstraints`, since it has a single usage.

Reply:
Sorry, this is an error from my part.